enterprise/management/src/main/java/org/neo4j/management/PageCache.java

```diff
@@ -57,4 +57,7 @@
                   "This number should be zero, or at least not growing, in a healthy database. " +
                   "Otherwise it could indicate drive failure, storage space, or permission problems." )
     long getEvictionExceptions();
+
+    @Description( "The percentage of used pages." )
+    double getUsageRatio();
```

Comment:
Can you get around the revapi complaint by giving this a default implementation? I think the NaN possibility should be mentioned here as well.

Reply:
I could probably get around it by providing a default implementation.